authentication/identity_providers/configuring-request-header-identity-provider.adoc

```diff
@@ -12,6 +12,8 @@ include::modules/identity-provider-overview.adoc[leveloffset=+1]
 
 include::modules/identity-provider-about-request-header.adoc[leveloffset=+1]
 
+include::modules/identity-provider-configuring-apache-request-header.adoc[leveloffset=+1]
```

Comment:
Being an example, this should be the latest subsection on this page

Reply:
Just to clarify, do you think this should come after the [Adding an identity provider to your clusters](https://docs.openshift.com/container-platform/4.2/authentication/identity_providers/configuring-request-header-identity-provider.html#add-identity-provider_configuring-request-header-identity-provider) subsection?

Reply:
Yes